src/test/groovy/graphql/nadel/RemovedFieldsTest.groovy

```diff
@@ -662,8 +661,69 @@ class RemovedFieldsTest extends StrategyTestHelper {
         errors[0].message.contains("removed field")
     }
 
+    def "inserts one error for a forbidden field in a list"() {
+        given:
+        GraphQLSchema overallSchema = TestUtil.schemaFromNdsl([Issues: '''
+        service Issues {
+            type Query {
+                issues: [Issue]
+            }
+ 
+            type Issue {
+                id: ID
+                relatedIssue: Issue                 
+                restricted: String
+            }
+        }
+        '''])
+        GraphQLSchema issueSchema = TestUtil.schema("""
+        type Query {
+            issues: [Issue]
+        }
+
+        type Issue {
+            id: ID
+            relatedIssue: Issue                 
+            restricted: String
+        }
+        """)
+        def query = "{issues {id restricted}}"
+
+        def hooks = new ServiceExecutionHooks() {
+            @Override
+            CompletableFuture<Optional<GraphQLError>> isFieldForbidden(NormalizedQueryField normalizedField, Object userSuppliedContext) {
```

Comment:
just a bit of a nit. You can do just 
`def hooks = createServiceExecutionHooksWithFieldRemoval(["restricted"])`

Reply:
Ah nice, fixed that up